tests/integration/kubernetes/confidential/k8s-confidential.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	( [ "${KATA_HYPERVISOR}" == "firecracker" ] || [ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-tdx" ] ) && skip "Test not supported yet."
+
+	pod_name="confidential-unencrypted-test"
+	container_name="confidential-unencrypted-container"
+	get_pod_config_dir
+
+	setup_unencrypted_confidential_pod
+}
+
+@test "Test unencrypted confidential container launch success and verify that we are running in a secure enclave." {
+	( [ "${KATA_HYPERVISOR}" == "firecracker" ] || [ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-tdx" ] ) && skip "Test not supported yet."
```

Comment:
In a common file, I'd add a list of confidential hypervisors and just check if ${KATA_HYPERVISOR}  is part of that list.

The initial list must include: qemu-sev, qemu-snp, qemu-tdx.

Reply:
Done.